README.md

```diff
@@ -1,6 +1,5 @@
 <div align="center">
-    <img src="https://raw.githubusercontent.com/neptune-ai/neptune-client/assets/readme/github-banner.jpeg" width="1500" />
-    &nbsp;
+    <img src="https://neptune.ai/wp-content/uploads/2023/08/Github-cover.png" width="1500" />
```

Comment:
Please upload all the images to `assets` branch. @domikoz probably knows the details of the process here.